### ISSUE TITLE
The version command does not support JSON output

### ISSUE THREAD

**Issue by 1977er** [restic/restic#4547]
Output of `restic version`
--------------------------

`restic 0.16.1 compiled with go1.21.3 on linux/amd64`

What should restic do differently? Which functionality do you think we should add?
----------------------------------------------------------------------------------

Something like

```
> restic version --json
[{"version":"0.16.1"},{"go_version":"1.21.3",{"target":"linux/amd64"}}]
```

What are you trying to do? What problem would this solve?
---------------------------------------------------------

Automation

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------

Great tool. Love it.

**Comment by MichaelEischer** [comment]
Sounds like a good idea. But the json output should be _far_ less nested and rather look similar to the following: `{"version":"0.16.1", "go_version":"1.21.3", "go_target":"linux/amd64"}`

**Comment by MichaelEischer** [comment]
Implemented in #4553
